code/game/objects/items/storage/pouch.dm

```diff
@@ -532,6 +532,46 @@
 	new /obj/item/storage/pill_bottle/tramadol(src)
 	new /obj/item/stack/medical/splint(src)
 
+/obj/item/storage/pouch/medkit/marine
+	name = "medkit pouch"
+	desc = "A standard use medkit pouch that can contain all kinds of stuff. This marine version has 6 slots."
+	icon_state = "medkit"
+	w_class = WEIGHT_CLASS_BULKY //does not fit in backpack
+	max_w_class = 4
```

Comment:
do not rewrite vars that are not changing compared to the parent.

Here you just need the storage slot and the desc, that's all